src/Components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,29 @@
+// import { useState } from 'react'
```

Comment:
Why is this commented?

Reply:
This will be used later in the code and was not useful in this build so commented it